src/json/serialization.cr

```diff
@@ -126,6 +126,28 @@ module JSON
   # field, and the rest of the fields, and their meaning, depend on its value.
   #
   # You can use `JSON::Serializable.use_json_discriminator` for this use case.
+  #
+  # ### `after_initialize` method
+  #
+  # `#after_initialize` is a method that runs after initialization and can be used
```

Comment:
I'm thinking "after initialization" is probably too imprecise.
Maybe this would be better?
```suggestion
  # `#after_initialize` is a method that runs after an instance is deserialized from JSON. It can be used
```

Reply:
I think this makes more sense from the readers perspective, plus I wasn't happy about using the word `initialize` and `initialization` back to back in the same sentence.